client/document/index.jsx

```diff
@@ -212,13 +215,17 @@ class Document extends Component {
 							__html: inlineScript,
 						} }
 					/>
-					{ i18nLocaleScript && ! useTranslationChunks && <script src={ i18nLocaleScript } /> }
+					{ i18nLocaleScript && ! useTranslationChunks && (
+						<script nonce={ inlineScriptNonce } src={ i18nLocaleScript } />
```

Comment:
TIl about the `nonce` [attribute](https://developer.mozilla.org/fr/docs/Web/HTML/Reference/Global_attributes/nonce)